jenkins/job-configs/kubernetes-jenkins-pull/testinfra-pull.yaml

```diff
@@ -55,6 +56,12 @@
             timeout: 600
             fail: true
     builders:
+        - ensure-upload-to-gcs-script:
```

Comment:
Does this download the current version from kubernetes/kubernetes? test-infra has a skewed/old version.